cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/services/ApplicationLifecycleService.java

```diff
@@ -347,28 +347,28 @@ public ApplicationWithPrograms deployApp(Id.Namespace namespace, @Nullable Strin
    * @param namespaceId the {@link Id.Namespace} under which all application should be deleted
    * @throws Exception
```

Comment:
Need to add a `@return` to describe what's being returned (there is no removal if the returned set is empty).